library/build.gradle

```diff
@@ -29,3 +32,24 @@ dependencies {
 }
 
 apply from: "${rootDir}/gradle/style.gradle"
+
+bintray {
+    user = System.getenv("BINTRAY_USER")
+    key = System.getenv("BINTRAY_KEY")
+    configurations = ["archives"]
+    pkg {
+        repo = "com.okta.android"
+        name = "okta-appauth"
+        desc = "Okta Authentication using AppAuth"
+        userOrg = "okta"
+        licenses = ["Apache-2.0"]
+        websiteUrl = "https://github.com/okta/okta-sdk-appauth-android"
+        vcsUrl = "https://github.com/okta/okta-sdk-appauth-android.git"
+        issueTrackerUrl = "https://github.com/okta/okta-sdk-appauth-android/issues"
+        version {
+            name = "0.1.0"
+            desc = "Okta Authentication using AppAuth"
+            vcsTag = "0.1.0"
```

Comment:
use the value in `"${rootProject.versionName}"` instead of redefining. Same for the `name` above.